azkaban-webserver/src/main/resources/azkaban/webapp/servlet/velocity/flowexecutionpanel.vm

```diff
@@ -201,7 +201,7 @@
       </div><!-- /modal -->
 
 #if (!$show_schedule || $show_schedule == 'true')
-  #parse ("azkaban/webapp/servlet/velocity/schedulepanel.vm")
+  #parse ("azkaban/webapp/servlet/velocity/${switchToPanelPage}")
```

Comment:
Is it less likely to make config mistakes if we make the config a binary switch?